docs/kusama-ledger.md

```diff
@@ -0,0 +1,170 @@
+- If asked, allow the manager on your device by pressing the right button.
+- Find Kusama in the app catalog.
+- Click the "Install" button of the app.
+- An installation window appears.
+- Your device will display "Processing…"
+- The app installation is confirmed.
+
+Please proceed to the usage instructions below.
+
+### Using the Developer Release
+
+> NOTE: These instructions are for development installation only. It is recommended to install the
+> application from Ledger Live unless you _know exactly what you're doing_.
+
+Instructions for downloading the prerelease binary from the GitHub releases is written [on the
+README][prerelease instructions] for the Kusama Ledger application GitHub repository.
+
+On the [releases page][] you can download the shell script `install_app.sh` and then make it
+executable in your shell by typing the command `chmod +x install_app.sh`.
+
+Using `install_app.sh` help command will show you the available options:
+
+```zsh
+$ ./install_app.sh --help
+Zondax Installer [Kusama-1.2011.1] [Warning: use only for test/demo apps]
+  load    - Load Kusama app
+  delete  - Delete Kusama app
+  version - Show Kusama app version
+```
+
+Next, you must make sure your Ledger device is plugged in and unlocked and you're using the latest
+firmware (1.6 at the time of writing). If everything is prepared, then type `install_app.sh load`
+and accept the prompts on your Ledger device to install the application.
+
+First it will prompt you to allow an unsafe manager. Navigate all the way to the left and click both
+buttons to allow the unsafe manager to install the Kusama app.
+
+After some processing time, the screen of your device will update to say "Install app Kusama".
+Again, navigate all the way to the right, verify the Identifier hash matches the one that is printed
+in your terminal. Click both buttons on "Perform Installation" to install the application. It will
+ask again for your Pin code and you should enter it in the device.
+
+At the end of the process you should the newly installed Kusama application on the device. Great
+job!
+
+## Using on Polkadot-JS Apps
+
+### Loading Your Account
+
+[Polkadot-JS Apps][] already has an integration with the Ledger application so that your device will
+work with the browser interface after installation. The functionality is currently gated behind a
+feature setting that you will need to turn on.
+
+In order to turn on the interoperability with the Kusama Ledger application, go to the "Settings"
+tab in Polkadot-JS Apps. Find the option for attachting Ledger devices and switch the option from
+the default "Do not attach Ledger devices" to "Attach Ledger via WebUSB".
+
+![](assets/ledger.png)
+
+Click "Save" to keep your settings.
+
+Now when you go to the "Accounts" tab you will see a new button that says "Query Ledger". Ensure
+that your Ledger device is unlocked and you have navigated into the Kusama application, then click
+this button.
```

Comment:
```suggestion
the query Ledger button.
```
I think we should be descriptive about buttons as some people might be visually impaired or have trouble with loading the image onto the screen.